server/json/q029.json

```diff
@@ -0,0 +1,29 @@
+{
+  "type": "q029",
+  "edits": [
+    {
+      "lar": {"loanId": "00001"},
+      "stateCode": "36",
+      "countyCode": "021",
+      "censusTract": "0004.02",
+      "recommendedMSA": "00",
+      "verification": ""
```

Comment:
But what about the case where they leave and come back? That initial render could have some `true`, some `false`, and some they haven't selected yet. If we're returning `false` for those we wouldn't know the difference.